Backpack/src/main/java/net/skyscanner/backpack/calendar/BpkCalendar.kt

```diff
@@ -36,8 +36,8 @@ open class BpkCalendar @JvmOverloads constructor(
   }
 
   private fun updateYearPill(year: Int) {
-    val currentYear = Calendar.getInstance().get(Calendar.YEAR)
-    year_pill_view.message = year.toString()
-    year_pill_view.visibility = if (currentYear == year) View.GONE else View.VISIBLE
+    val yearCalendar = LocalDate.of(year, 1, 1)
```

Comment:
This should be `LocalDate.now()`

Reply:
Actually, no need to keep track of month, day here so just going to use `YearMonth.now()` instead and simply assign the parameters `year: Int` as the message